bindings/rust/standard/bench/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "bench"
 version = "0.1.0"
 edition = "2021"
+publish = false
```

Comment:
`publish` was set to false in `bench` so that the `--ignore-private` flag would skip minimum version checks for it.